### DISCUSSION TITLE
BEDPE

### DISCUSSION THREAD

**Discussion by LuisSoares** [GMOD/jbrowse-components#3140] [Q&A]
Is there a away to display BEDPE as tracks in jbrowse2 using the ArcRenderer? Alternatively is it possible to change the colors of the arcs displayed in Linear Arc Display?

**Comment by garrettjstevens** [comment]
JBrowse is not able to display BEDPE files currently. There has been some discussion of a potential BEDPE parser here: https://github.com/GMOD/jbrowse-components/issues/306

As for the arc colors, if you go into the track's "Settings" (you may have to make a copy of the track first), you can change the color of the display renderer as shown here:

![](https://files.gitter.im/600b35cbd73408ce4ff97fd2/msPh/image.png)

If you're editing your config.json, the equivalent would be this:

```json
{
  "type": "FeatureTrack",
  "trackId": "dbsuper",
  "name": "dbSUPER adipose enhancers",
  "assemblyNames": ["hg19"],
  "adapter": {
    "type": "BedTabixAdapter",
    "bedGzLocation": {
      "uri": "https://s3.amazonaws.com/jbrowse.org/genomes/hg19/dbSUPER/adipose_interactions.sorted.bed.gz"
    },
    "index": {
      "location": {
        "uri": "https://s3.amazonaws.com/jbrowse.org/genomes/hg19/dbSUPER/adipose_interactions.sorted.bed.gz.tbi"
      }
    }
  },
  "displays": [
    {
      "type": "LinearArcDisplay",
      "displayId": "LinearBasicDisplay",
      "renderer": {
        "type": "ArcRenderer",
        "color": "rgb(248, 118, 109)"
      }
    }
  ]
}
```
